README.md

```diff
@@ -7,6 +7,12 @@ Management Server component of uProxy for Orgs (UfO)
 [![Travis Status](https://travis-ci.org/uProxy/ufo-management-server-flask.svg)](https://travis-ci.org/uProxy/ufo-management-server-flask)
 [![Code Health](https://landscape.io/github/uProxy/ufo-management-server-flask/master/landscape.svg?style=flat)](https://landscape.io/github/uProxy/ufo-management-server-flask/master)
 
+## Feedback
+
+If you would like to report a problem or request a new feature with the management server or uProxy overall, feel free to use the issue tracker on [this repo](https://github.com/uProxy/ufo-management-server-flask/issues) or on the [uProxy issue tracker](https://github.com/uProxy/uproxy/issues). Wherever you file, please include as much pertinent information as possible, such as the version of the management server being used, date and time the issue was encountered, steps to reproduce the issue, what browser and platform the issue was encountered upon, etc.
```

Comment:
:+1:   Awesome, the template will help the user and also help us.  :)  So, if that is in place, I would just remove the whole line below, as everything we need from the user will be embedded in the issue.

"Wherever you file, please include as much pertinent information as possible, such as the version of the management server being used, date and time the issue was encountered, steps to reproduce the issue, what browser and platform the issue was encountered upon, etc."